site/downloads.markdown

```diff
@@ -71,6 +71,18 @@ Alternatively, many operating systems provide GHC, cabal and Stack through their
 
 **Do not use the Haskell development tools provided by Arch, they are broken.** For more information see [[1]](https://dixonary.co.uk/blog/haskell/cabal-2020) [[2]](https://stackoverflow.com/questions/65643699/what-is-the-suggested-way-of-setting-up-haskell-on-archlinux/65644318#65644318).
 
+#### NixOS
+
+[](This installation method is owned by @maralorn)
```

Comment:
Regrettably it turns out that this trick for "Markdown comments" doesn't play well with our "link checker" than checks all links exist. The "pseudo link" `This installation method is owned by @maralorn` doesn't exist, so link checker fails. I think we should probably switch to:

```suggestion
<!-- This installation method is owned by @maralorn -->
```